wrapper/virt_v2v_wrapper.py

```diff
@@ -487,6 +498,32 @@ def main():
                                                   host.get_uid(),
                                                   host.get_gid())
 
+        if 'luks_keys_vault' not in data:
+            data['luks_keys_vault'] = os.path.join(
+                os.environ['HOME'],
+                '.v2v_luks_keys_vault.json'
+            )
+        if os.exists(data['luks_keys_vault']):
+            file_stat = os.stat(data['luks_keys_vaul'])
+            if file_stat.st_uid != host.get_uid():
+                hard_error('LUKS keys vault does\'nt belong to'
+                           'user running virt-v2v-wrapper')
+            if not bool(file_stat.st_mode & stat.S_IRWXO):
```

Comment:
This cast is ugly. I would prefer `if file_stat.st_mode & stat.S_IRWXO > 0:`

Reply:
Done